settings.h

```diff
@@ -712,7 +712,9 @@ typedef struct {
     float dual_axis_offset;
     float homing_seek_rate;
     float homing_feed_rate;
+#if ENABLE_JERK_ACCELERATION 
```

Comment:
Remove this #if/#endif, it will trigger a settings reset when enabling/disabling jerk.

Reply:
Done.